### PR TITLE
Custom Scheme Registration

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
* **Working with local content in WebView2 apps** > **Loading local content by handling the WebResourceRequested event** > **Custom scheme registration** - new section
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/working-with-local-content?branch=pr-en-us-3121#custom-scheme-registration

This PR fixes https://github.com/MicrosoftDocs/edge-developer/issues/3120.
This PR also adds some blank lines in the .md source file.

AB#49821215

### CODE REVIEW

File: microsoft-edge/webview2/concepts/working-with-local-content.md [hunk 1]
```diff
@@ -32,17 +32,25 @@ The various ways of loading local content into a WebView2 control support the fo
 | Additional web resources | ✔️ | ❌ | ✔️  | ✔️ |
 | Additional web resources resolved in WebView2 process | ✔️ | ❌ | ✔️ | ❌ |
 
+<!-- todo: add table row for custom scheme registration? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/working-with-local-content.md:35]: add table row for custom scheme registration?
@aluhrs13 
rendered article section:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/working-with-local-content?branch=pr-en-us-3121&tabs=dotnetcsharp#selecting-an-approach

File: microsoft-edge/webview2/concepts/working-with-local-content.md [hunk 2]
```diff
@@ -786,6 +794,12 @@ HRESULT AppWindow::WebResourceRequestedEventHandler(
 ---
 
 
+<!-- ====================================================================== -->
+## Custom scheme registration
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/working-with-local-content.md:798]: Will the added content be an h2 section?  
Will the h2 section be at end of article?
How does custom scheme registration fit in w/ rest of article?
@aluhrs13

Comment (aluhrs13) [reply microsoft-edge/webview2/concepts/working-with-local-content.md:798]: Probably should be an H2, @yildirimcagri-msft is probably the right person to work with on the details of this, I'm not super familiar with using it myself.

File: microsoft-edge/webview2/concepts/working-with-local-content.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/working-with-local-content.md:35]: no add'l col needed b/c 4th approach includes the new h4 section's approach.

File: microsoft-edge/webview2/concepts/working-with-local-content.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/working-with-local-content.md]: new section is h4 within existing h2 section.

File: microsoft-edge/webview2/concepts/working-with-local-content.md (hunk 1 above)
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/working-with-local-content.md:35]:
```suggestion
```

**Review by aluhrs13 — APPROVED** [review]

**Pull request merged**